tests/testutils/_primer/test_primer.py

```diff
@@ -36,12 +36,6 @@ def test_primer_launch_bad_args(args: list[str], capsys: CaptureFixture) -> None
     assert "usage: Pylint Primer" in err
 
 
-@pytest.mark.skipif(
```

Comment:
This is why I didn't see it originally. These tests take 2 seconds, let's just run them on all platforms.